5.8.0-rc.19/kubectl-provision

```diff
@@ -1173,8 +1173,9 @@ function print_vault_keys {
 }
 
 # NOTE: verbose will only show up if you execute with "V=1 ./kubectl-kubectl"
+set +e
```

Comment:
Why is this needed? The "|| verbose" should ensure that this does not error out.  Have you experienced any aborts at this line?

Reply:
Both M1 macbooks using `zsh` - but I wouldn't expect that to make a difference

Reply:
```
➜  ~ V=1 kubectl-provision --version
➜  ~
```
```
➜  ~ echo $?
0
```

Reply:
```
➜  SH git:(release/5.8.0-rc.20_branch) V=1 ./kubectl-provision --debug --version
➜  SH git:(release/5.8.0-rc.20_branch)
```

with set -e
```
➜  SH git:(release/5.8.0-rc.20_branch) ✗ V=1 ./kubectl-provision --debug --version
+ : '
Access to this file is granted under the SCONE COMMERCIAL LICENSE V1.0

Any use of this product using this file requires a commercial license from scontain UG, www.scontain.com.

Permission is also granted  to use the Program for a reasonably limited period of time  (but no longer than 1 month)
for the purpose of evaluating its usefulness for a particular purpose.

THERE IS NO WARRANTY FOR THIS PROGRAM, TO THE EXTENT PERMITTED BY APPLICABLE LAW. EXCEPT WHEN OTHERWISE STATED IN WRITING
THE COPYRIGHT HOLDERS AND/OR OTHER PARTIES PROVIDE THE PROGRAM "AS IS" WITHOUT WARRANTY OF ANY KIND, EITHER EXPRESSED OR IMPLIED,
INCLUDING, BUT NOT LIMITED TO, THE IMPLIED WARRANTIES OF MERCHANTABILITY AND FITNESS FOR A PARTICULAR PURPOSE.

THE ENTIRE RISK AS TO THE QUALITY AND PERFORMANCE OF THE PROGRAM IS WITH YOU. SHOULD THE PROGRAM PROVE DEFECTIVE,
YOU ASSUME THE COST OF ALL NECESSARY SERVICING, REPAIR OR CORRECTION.

IN NO EVENT UNLESS REQUIRED BY APPLICABLE LAW OR AGREED ON IN WRITING WILL ANY COPYRIGHT HOLDER, OR ANY OTHER PARTY WHO MAY
MODIFY AND/OR REDISTRIBUTE THE PROGRAM AS PERMITTED ABOVE, BE LIABLE TO YOU FOR DAMAGES, INCLUDING ANY GENERAL, SPECIAL,
INCIDENTAL OR CONSEQUENTIAL DAMAGES ARISING OUT OF THE USE OR INABILITY TO USE THE PROGRAM INCLUDING BUT NOT LIMITED TO LOSS
OF DATA OR DATA BEING RENDERED INACCURATE OR LOSSES SUSTAINED BY YOU OR THIRD PARTIES OR A FAILURE OF THE PROGRAM TO OPERATE
WITH ANY OTHER PROGRAMS), EVEN IF SUCH HOLDER OR OTHER PARTY HAS BEEN ADVISED OF THE POSSIBILITY OF SUCH DAMAGES.

Copyright (C) 2022-2023 scontain.com
'
+ set -e
+ export K_PROVISION_VERSION=5.8.0-rc.20
+ K_PROVISION_VERSION=5.8.0-rc.20
+ export 'K_SGX_TOLERATIONS=--accept-configuration-needed --accept-group-out-of-date --accept-sw-hardening-needed'
+ K_SGX_TOLERATIONS='--accept-configuration-needed --accept-group-out-of-date --accept-sw-hardening-needed'
+ export K_MRSIGNER_DB=195e5a6df987d6a515dd083750c1ea352283f8364d3ec9142b0d593988c6ed2d
+ K_MRSIGNER_DB=195e5a6df987d6a515dd083750c1ea352283f8364d3ec9142b0d593988c6ed2d
+ export K_MRSIGNER_SCONTAIN=b6c14ae06a05284253b548038e269f8225718bc0856828f491ed85174ccc2979
+ K_MRSIGNER_SCONTAIN=b6c14ae06a05284253b548038e269f8225718bc0856828f491ed85174ccc2979
+ export K_ISVPRODID=41316
+ K_ISVPRODID=41316
+ export K_ISVSVN=5
+ K_ISVSVN=5
+ export 'RED=\e[31m'
+ RED='\e[31m'
+ export 'BLUE=\e[34m'
+ BLUE='\e[34m'
+ export 'ORANGE=\e[33m'
+ ORANGE='\e[33m'
+ export 'NC=\e[0m'
+ NC='\e[0m'
+ export SCONTAIN_IMAGE_REPO=registry.scontain.com/scone.cloud
+ SCONTAIN_IMAGE_REPO=registry.scontain.com/scone.cloud
+ export DEFAULT_MANIFESTS_URL=https://raw.githubusercontent.com/scontain/manifests/main
+ DEFAULT_MANIFESTS_URL=https://raw.githubusercontent.com/scontain/manifests/main
+ export DEFAULT_DCAP_KEY=aecd5ebb682346028d60c36131eb2d92
+ DEFAULT_DCAP_KEY=aecd5ebb682346028d60c36131eb2d92
+ export CAS_CLIENT_PORT=8081
+ CAS_CLIENT_PORT=8081
+ SIGNER=5BCAD31DCC8D5D722B7B7ABD2EBE04E7CC816D32
+ trap 'last_command=$current_command; current_command=$BASH_COMMAND' DEBUG
++ last_command=
++ current_command='trap '\''if [ $? -ne 0 ]; then echo -e "${RED}\"${last_command}\" command failed - exiting.${NC}"; if [ $SERVICE_PID != 0 ] ; then kill $SERVICE_PID ; fi ; fi'\'' EXIT'
+ trap 'if [ $? -ne 0 ]; then echo -e "${RED}\"${last_command}\" command failed - exiting.${NC}"; if [ $SERVICE_PID != 0 ] ; then kill $SERVICE_PID ; fi ; fi' EXIT
++ last_command='trap '\''if [ $? -ne 0 ]; then echo -e "${RED}\"${last_command}\" command failed - exiting.${NC}"; if [ $SERVICE_PID != 0 ] ; then kill $SERVICE_PID ; fi ; fi'\'' EXIT'
++ current_command='source "$CONFIG_FILE" 2> /dev/null'
+ source ''
```